src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -144,7 +148,7 @@ public override void OnEventWritten(string name, object payload)
             // We are relying here on ASP.NET Core to set duration before writing the stop event.
             // https://github.com/dotnet/aspnetcore/blob/d6fa351048617ae1c8b47493ba1abbe94c3a24cf/src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs#L449
             // TODO: Follow up with .NET team if we can continue to rely on this behavior.
-            this.httpServerDuration.Record(Activity.Current.Duration.TotalMilliseconds, tags);
+            this.httpServerDuration.Record(Activity.Current.Duration.TotalSeconds, tags);
```

Comment:
Java gates this change with the `OTEL_SEMCONV_STABILITY_OPT_IN` setting. Maybe we should too since this is potentially disruptive to end users and backends.

Unfortunately, it's not explicitly stated that this change should be gated, but maybe it should be... @trask what do you think?

Reply:
within the scope of HTTP semconv, I think(?) this is implied by the spec, since unit `seconds` is part of the frozen http semconv for `http.server.request.duration`

Reply:
> I think(?) this is implied by the spec

This was my original interpretation... but I also can understand how someone might have a different interpretation since the words from the [warning text](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/README.md#semantic-conventions-for-http) are very attribute-centric.

I'd be happy to take a stab at some clarifying verbiage to the text if folks would find that helpful.

Reply:
> the words from the [warning text](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/README.md#semantic-conventions-for-http) are very attribute-centric.

ah, thx, I see the confusion now

> I'd be happy to take a stab at some clarifying verbiage to the text if folks would find that helpful.

that would be great